README.md

```diff
@@ -132,6 +132,11 @@ Table of Contents
     * .tech Domain Name - Free to use for one year
     * Full, detailed list - https://www.alibabacloud.com/campaign/free-trial
 
+
+  * [DigitalOcean](https://www.digitalocean.com/)
+    * [GitHub Student Developer Pack for free $50 DigitalOcean Credit](https://www.digitalocean.com/github-students/)
```

Comment:
this is probably a bt low to bother with tbh, $50 doesnt get you far not over a year